configobject/statesync/overdue.go

```diff
@@ -156,32 +156,43 @@ func updateOverdue(super *supervisor.Supervisor, objectType string, counter *uin
 
 // updateOverdueInDb sets objectType_state#is_overdue for ids to overdue.
 func updateOverdueInDb(super *supervisor.Supervisor, objectType string, observer prometheus.Observer, ids []interface{}, overdue bool) {
-	placeholders := make([]string, 0, len(ids))
-	for len(placeholders) < cap(placeholders) {
-		placeholders = append(placeholders, "?")
-	}
+	for _, c := range utils.ChunkInterfaces(ids, 1000) {
+		go func(chunk []interface{}) {
```

Comment:
You're not waiting for this to finish. Are you sure that's a good idea?

Reply:
I'm not sure why we should wait here. Any benefits?

Reply:
Okay, this is a reason:
https://github.com/Icinga/icingadb/blob/b20cb2e4ed4282335bd529952144a8241d24e5a1/configobject/statesync/overdue.go#L150

Anything else?

Reply:
No, I've already requested all changes I'd like to see.